build.gradle

```diff
@@ -56,6 +56,8 @@ compileTestKotlin {
 ext {
 	assertkVersion = '0.13'
 	jacksonVersion = '2.9.8'
+	junitJupiterVersion = '5.4.0'
```

Comment:
There is already `5.4.1`: https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter-engine

However I guess for `master` we can just go ahead with the `5.5.0-M1`.

Should be available for our GA in the future anyway.